### ISSUE TITLE
Crash on any file that is not main.rs

### ISSUE THREAD

**Issue by PhilipK** [willcrichton/flowistry#50]
Opens window with crash info when toggled in a file that is not main.rs, but will work in main.rs.

Example output:
```
Flowistry could not run because your project failed to build with error:
thread 'main' panicked at 'Could not find target for path: /home/philip/src/BoilR/src/ui/uiapp.rs', crates/flowistry_ide/src/bin/cargo-flowistry.rs:180:10
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

This is the project I have tested on:
https://github.com/PhilipK/BoilR

**Comment by PhilipK** [comment]
I have tried a couple other projects and it works on them.
Let me know if there is anything I can do to try and debug why it is crashing.

**Comment by willcrichton** [comment]
I will check out the repo, thanks for the report @PhilipK.

**Comment by willcrichton** [comment]
I have fixed this issue, the 0.5.21 release should work on your repo now.
